docs/REST API/rest-api.md

```diff
@@ -11396,6 +11403,7 @@ By default, the user will be forced to reset its password upon first login.
 | name        | string  | body | **Required**. The user's full name or nickname.                                                                                                                                                                                                                                                                                                          |
 | password    | string  | body | The user's password (required for non-SSO users).                                                                                                                                                                                                                                                                                                        |
 | sso_enabled | boolean | body | Whether or not SSO is enabled for the user.                                                                                                                                                                                                                                                                                                              |
+| mfa_enabled | boolean | body | _Available in Fleet Premium._ Whether or not the user must click a magic link emailed to them to log in, after they successfully enter their username and password. Incompatible with SSO and API-only users. |
```

Comment:
@iansltx how do we handle a user attempting to set this when `sso_enabled` is `true`? Are we throwing an error?

Reply:
Yeah, we're erroring if SSO + MFA. For API only + MFA we will create the user but will fail on pulling the session, so if an admin really wants an API user with MFA we won't stop them, but it's batteries not included.